stdpopsim/catalog/CanFam/__init__.py

```diff
@@ -106,8 +107,16 @@
     generation_time=3,
     generation_time_citations=[
         # Everyone uses 3 years because everyone else uses it.
-        # It's likely higher, at least in wolves:
-        # https://pubs.er.usgs.gov/publication/70187564
+        # It's likely higher, at least in wolves: https://
+        # digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1304&context=usgsnpwrc
```

Comment:
Long URLs and line length limits don't interact well. It's a pity this doesn't have a DOI. How about using this shorter URL instead? https://science.sciencemag.org/content/352/6290/1228/tab-e-letters

Reply:
Yes, that one should work as well! The more I look at it the original link should work as well! I was just a tad confused why it was on the USGS website and not Science's..

Reply:
I updated it with the new link1